styles/index.css

```diff
@@ -1,3 +1,62 @@
+.overlay-container {
+  width: 100%;
+  height: 100vh;
+  position: relative;
+}
+.overlay-text {
+  position: absolute;
+  color: rgb(234, 234, 233);
+  top: 35%;
+  left: 5%;
```

Comment:
We're now using a 'virtual' 1500px max-width. So here is a layout challenge:
Try to set the text and button so that it 'meets' the bottom. With a margin like 64px
`left:` should be so that it fit with the max-with centered on the left side

Reply:
So, I removed the top:35% and added bottom:64px for the overlay-text so the container for text + button is 64px from the bottom of the container/background image.
I do not understand your left side instructions. I adjusted the padding-left: 64px and it looks more similar to your design but not sure if its what you asked for.

Reply:
>text + button is 64px from the bottom of the container/background image

that sounds good 👍 
>understand your left side instructions..

It means that as now the page has a max-width of 1500px with auto margin on left and right
at any given view width there is an appropriate value of margin left and margin right equally
the absolute positioning `left: ..px` should match `margin-left`

Reply:
![Uploading CleanShot 2024-10-25 at 13.25.58.png…]()